fhir/fhir-connector/fhir-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/fhir/fhirConnectorIntegrationTest.java

```diff
@@ -0,0 +1,383 @@
+package org.wso2.carbon.connector.integration.test.fhir;
+/*
+ *
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.connector.integration.test.base.ConnectorIntegrationTestBase;
+import org.wso2.connector.integration.test.base.RestResponse;
+
+
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/pull/558/commits/905699d38bfdd610426958914192be43b8018753